src/test/java/com/atlassian/db/replica/api/TestConsistency.java

```diff
@@ -45,21 +29,12 @@ public void shouldRefreshCommitted() throws SQLException {
         verify(consistency).write(any());
     }
 
-    private DualConnection.Builder dualConnectionBuilder(ConnectionProviderMock connectionProvider,
-                                                         ReplicaConsistency consistency) {
-        DualConnection.Builder builder = DualConnection.builder(connectionProvider, consistency);
-        if (shouldUseCompatibleWithPreviousVersion) {
-            builder.compatibleWithPreviousVersion();
```

Comment:
Each release we'll add and remove the boilerplate. With `FeatureDecisions` we wouldn't have it.
But let's stick with it for a few more releases and let's see.

Reply:
I agree, i think it was mentioned and highlighted in the original commit of `compatibleWithPreviousVersion`.

Let's see.

Reply:
> Each release we'll add and remove the boilerplate. With FeatureDecisions we wouldn't have it.
But let's stick with it for a few more releases and let's see.

I see it as an advantage. It forces us to do the cleanups as we go instead to accumulate FFs as we do in the product 😉 .